freedomofpress/securedrop
# Back out functionality requiring libjpeg-dev apt dependency

## Status

Ready for review

## Description of Changes

Fixes #3375. Fixes #3316. Re-opens #2807 ("Re-opens" is not a keyword so the merger of this PR should be sure the ticket is indeed re-opened).

Changes proposed in this pull request:
* Remove functionality requiring `libjpeg-dev` and thus `Pillow`. The functionality change is that we won't be able to accept JPG/JPEG logos (a lesser issue) and we won't be able to resize images server-size (a greater issue, as such this part is unfortunate). Note that we could potentially implement a roundabout way to do this, but I advocate we just fix the fact that we can't safely add `libjpeg-dev`, and then bring in the changes again. 

🌏Note 🌏 This involves a string change, which I apologize for. I could not find a way to reuse old strings on the interface that would not be very awkward. Suggestions are welcome here. 

## Testing

0. Provision staging on this branch. 
1. Try to submit a PNG, observe success.
2. Try to submit a JPG or JPEG, observe failure.

## Deployment

This can be safely deployed to instances

## Checklist

### If you made changes to the server application code:


### If you made changes to the system configuration:


### If you made changes to documentation:
